windows/hid.c

```diff
@@ -897,38 +976,26 @@ struct hid_hotplug_callback {
 	struct hid_hotplug_callback *next;
 };
 
-static struct hid_hotplug_context {
-	/* Win32 notification handle */
-	HCMNOTIFICATION notify_handle;
-
-	/* HIDAPI unique callback handle counter */
-	hid_hotplug_callback_handle next_handle;
-
-	/* Linked list of the hotplug callbacks */
-	struct hid_hotplug_callback *hotplug_cbs;
-
-	/* Linked list of the device infos (mandatory when the device is disconnected) */
-	struct hid_device_info *devs;
-} hid_hotplug_context = {
-	.notify_handle = NULL,
-	.next_handle = 1,
-	.hotplug_cbs = NULL,
-	.devs = NULL
-};
-
-DWORD WINAPI hid_internal_notify_callback(HCMNOTIFICATION notify, PVOID context, CM_NOTIFY_ACTION action, PCM_NOTIFY_EVENT_DATA event_data, DWORD event_data_size)
+DWORD WINAPI hid_internal_notify_callback(HCMNOTIFICATION notify,
```

Comment:
revert

Reply:
agree
looks like you're using some kind of IDE wich does its own formatting - that better be disabled, as it introduces too much code-style changes and makes the code inconsistent

unfortunately we don't have a clang-format spec for this repo (yet) so we have to keep the consistance/readability manually